docker-compose.yaml

```diff
@@ -19,7 +19,7 @@ x-common-environment: &common-environment
   IPFS_BASIC_AUTH_SECRET: ${IPFS_BASIC_AUTH_SECRET:-""}
   QUEUE_HIGH_WATER: 1000
   CHAIN_ENVIRONMENT: dev
-  DEBUG: true
+  DEBUG: 'true'
```

Comment:
Boolean values must be quoted, or the yaml parser can convert them to incorrect values and Docker gets confused.